src/base/SwapRouter.sol

```diff
@@ -22,24 +29,10 @@ abstract contract SwapRouter is UniV3Immutables, Payments, IUniswapV3SwapCallbac
     uint160 internal constant XOR_SQRT_RATIO =
         (4295128739 + 1) ^ (1461446703485210103287273052203988822378723970342 - 1);
 
-    /// @inheritdoc IUniswapV3SwapCallback
-    function uniswapV3SwapCallback(int256 amount0Delta, int256 amount1Delta, bytes calldata data) external {
-        // Only accept callbacks from an official Uniswap V3 pool
-        address pool = CallbackValidation.verifyCallbackCalldata(factory, data);
-        if (amount0Delta > 0) {
-            address token0;
-            assembly {
-                token0 := calldataload(data.offset)
-            }
-            pay(token0, address(this), pool, uint256(amount0Delta));
-        } else {
-            address token1;
-            assembly {
-                token1 := calldataload(add(data.offset, 0x20))
-            }
-            pay(token1, address(this), pool, uint256(amount1Delta));
-        }
-    }
+    /// @notice Deterministically computes the pool address given the pool key
+    /// @param poolKey The pool key
+    /// @return pool The contract address of the pool
+    function computeAddressSorted(PoolKey memory poolKey) internal view virtual returns (address pool) {}
```

Comment:
```suggestion
    function computeAddressSorted(PoolKey memory poolKey) internal view virtual returns (address pool);
```

Reply:
Thank you, Shuhui! I was trying to achieve this but was tricked by VS code into thinking I have to provide a dummy implementation.